### PR TITLE
Add upsell notification to builder tabs

### PR DESCRIPTION
**Pull request by UVLabs**
### Summary
Adds upsell notification to all builder tabs except Elementor builder tab

### Test instructions

- The upsell notice should  whether a user is not using a valid license
- The notice should show in all tabs except elementor

Closes Codeinwp/neve-pro-addon#1739

### CODE REVIEW

File: assets/src/scss/_media-queries.scss [hunk 1]
```diff
@@ -17,17 +17,19 @@
...
 }
 
-
 @media (min-width: #{$desktop-large}) {
 	@include general--desktop-large();
 	@include card--desktop-large();
 	@include ob-editor-tabs--desktop-large();
 }
+
+@media( max-width: #{$tablet}){
```
Comment (abaicus) [inline assets/src/scss/_media-queries.scss:33]: @UVLabs The CSS for this is mobile-first and only uses min-width media queries. Let's respect the convention and to the styles like that 👍🏻

File: assets/src/Components/StarterSites/Filters.js [hunk 2]
```diff
@@ -119,6 +120,13 @@ const Filters = ( {
 						onlyProSites={ onlyProBuilders }
 						count={ counted.builders }
 					/>
+					{ ! tiobDash.isValidLicense && editor !== 'elementor' && (
+					<Notification
+						data={tiobDash.upsellNotifications['upsell_1'] }
+						editor={editor}
+					/>
+					)
+					}
```
Comment (abaicus) [inline assets/src/Components/StarterSites/Filters.js:129]: Please enable eslint formatting on save on your end or run `eslint --fix` on this file 🦭

**Review by abaicus — CHANGES_REQUESTED** [review]
Thanks, @UVLabs! 👍🏻 For the most part, it looks good, except what I mentioned in the review comments. 

@cristian-ungureanu I'm not sure if we should add an external link icon to the button.

**Comment by cristian-ungureanu** [comment]
@UVLabs Andrei is right, the buttons should not have the external link icon.

**Comment by abaicus** [comment]
@cristian-ungureanu I was saying that it doesn't have, but it goes to an external link, so I think maybe we should be adding one 👀

**Comment by cristian-ungureanu** [comment]
@abaicus I was looking at other buttons in the dashboard that are leading to an external link and took the example from there: https://vertis.d.pr/qXWv5q, https://vertis.d.pr/RgPvj9, https://vertis.d.pr/35ZLmZ ... in this case, we should update other buttons too.

**Comment by abaicus** [comment]
Ok, makes sense. 👍🏻

**Comment by cristian-ungureanu** [comment]
@UVLabs let's add the external icon in the button too and I will create another task to update the others. 
Also, let's add this notice in Elementor's tab too.

**Comment by UVLabs** [comment]
@cristian-ungureanu @abaicus I've made the changes here

**Comment by cristian-ungureanu** [comment]
@UVLabs can you have a look and fix the changes here?

**Comment by ineagu** [comment]
@cristian-ungureanu The original description was that the notice would be shown only for tabs where there are exclusively pro sites: https://github.com/Codeinwp/neve-pro-addon/issues/1739, however I see it as well on the gutenberg tab. Which is not ideal, especially at the first view.

**Pull request merged**